src/Sylius/Bundle/GridBundle/Form/Type/Filter/SelectFilterType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Form\Type\Filter;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class SelectFilterType extends AbstractType
+{
+
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver
+            ->setDefault('placeholder', 'sylius.ui.all')
+            ->setAllowedTypes('choices', ['array']
+            )
```

Comment:
```suggestion
        );
```

And the line below should be removed (the one with `;`)